File: app/src/main/java/fr/erictruong/training/material/ui/fragment/TooltipsFragment.java
package fr.erictruong.training.material.ui.fragment;

import fr.erictruong.training.material.R;
import fr.erictruong.training.material.ui.card.Card;
import fr.erictruong.training.material.ui.card.HeadlineBodyCard;

import java.util.ArrayList;
import java.util.List;

public class TooltipsFragment extends MaterialTrainingNavigationDrawerFragment {

    @Override
    protected List<Card> populateDataset() {
        List<Card> dataset = new ArrayList<>();
        dataset.add(new HeadlineBodyCard(0, Card.TYPE_PRIMARY_HEADLINE_BODY_2, getString(R.string.fragment_tooltips), getString(R.string.fragment_tooltips_txt)));
        return dataset;
    }
}

File: app/src/main/java/fr/erictruong/material/training/ui/components/tooltips/TooltipsFragment.kt
package fr.erictruong.material.training.ui.components.tooltips

import android.os.Bundle
import android.view.View
import fr.erictruong.android.cards.item.DisplayBodyItem
import fr.erictruong.android.cards.item.MaterialCardItem
import fr.erictruong.material.training.R
import fr.erictruong.material.training.ui.core.BaseCardFragment
import java.util.*

class TooltipsFragment : BaseCardFragment() {

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val cards = ArrayList<MaterialCardItem>()
        // display1_primary body2
        cards.add(DisplayBodyItem(display = getString(R.string.tooltips), body = getString(R.string.tooltips_txt)))
        // TODO: Add cards and stuff.

        adapter.swapData(cards)
    }
}
